### DISCUSSION TITLE
The problem of not changing the photo

### DISCUSSION THREAD

**Discussion by mostafamiri74** [MurhafSousli/ngx-gallery#605] [General]
Hello. I want a simple product photo gallery and I used several methods of this library. Everything works correctly (including the number on the photo and thumbnails), but the gallery photo does not change.

**Comment by MurhafSousli** [comment]
Can you add a stackblitz reproduction for you issue?

**Reply by mostafamiri74** [reply]
No, because I deleted the codes. I can tell the steps of doing my work. After installing the package, I put the code related to the token in the provider in app.module. Then I did the same as the codes in this address. https://ngx-gallery.netlify .app/#/gallery
This gallery does not work inside this page either.

**Reply by MurhafSousli** [reply]
I cannot help without reproduction because everything works on my end, if you need help, you need to reproduce your issue so I can have a look at it.
